test/libsolidity/semanticTests/using/module_2.sol

```diff
@@ -0,0 +1,33 @@
+==== Source: A ====
+library L {
+    function id(uint x) internal pure returns (uint) {
+        return x;
+    }
+    function one_ext(uint) pure external returns(uint) {
+        return 1;
+    }
+    function empty() pure internal {
+    }
+
+}
+
+==== Source: B ====
+import "A" as M;
+
+contract C {
+    using M.L for uint;
+	function f(uint x) public pure returns (uint) {
+        return x.id();
+    }
+    function g(uint x) public pure returns (uint) {
+        return x.one_ext();
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// library: L
```

Comment:
@cameel This fails currently, I think it is expecting `A.L`. However, isoltest seems to not support separating with dots.

Reply:
True, but I think it was just never implemented (rather than broken by #12073). The thing that the PR changed is that you can now put the source unit name in front of the library name..

Should not be hard to implement though. Changes are likely needed in the same places as in #12073. The main thing was the [change in `TestFileParser.cpp`](https://github.com/soroosh-sdi/solidity/commit/816d8021e4f57b1b32eb3ce1fbfa421a585a0f0f#diff-ca8456220457a41b507d3968dbffa4ed8b777a91653fec0eedd4985fd9bf5606). The rest is mostly adjusting tests.

Reply:
Oh, wait. Even if you import library `L` from module `A` its fully qualified name does not become `A.L`. It's always qualified with the source unit name. So on the CLI you would use `A:L`. In soltest you need `"A".L` because source unit names can contain arbitrary characters and it was too hard to get the parser to allow that without something like quoting or a special prefix.

Reply:
> So no need to handle dots at all.

What do you mean? Changing to "A".L is also unsupported by isoltest currently.